SQL.test.js

```diff
@@ -194,7 +195,7 @@ test('SQL helper - build complex query with append and glue', t => {
   )
   t.equal(
     sql.debug,
-    'TEST QUERY glue pieces FROM v1 = \'v1\' , v2 = \'v2\' , v3 = \'v3\' , v4 = \'v4\' , v5 = \'v5\' WHERE v6 = \'v6\' AND v7 = \'v7\''
+    "TEST QUERY glue pieces FROM v1 = 'v1' , v2 = 'v2' , v3 = 'v3' , v4 = 'v4' , v5 = 'v5' WHERE v6 = 'v6' AND v7 = 'v7'"
```

Comment:
why was this changed? I would not expect it to be.

Reply:
because standard was updated and it's standard who decides how quotes are used within strings, which changed across standard versions